### PR TITLE
Fixed UI for all forms in responsive view

### PR DESCRIPTION
**Pull request by enigmaticmahesh**
## Summary

Made forms responsive for mobile views.
PR for Issue #1582

### CODE REVIEW

File: src/containers/Search/Search.module.css [hunk 1]
```diff
@@ -57,3 +57,13 @@
 .FormSearch > div:nth-of-type(5) {
   margin-right: 18px;
 }
+
+@media (max-width:768px) {
+  .Form > div:nth-of-type(8), .Form > div:nth-of-type(9) {
```
Comment (kurund) [inline src/containers/Search/Search.module.css:62]: This approach seems very fragile

File: src/containers/SettingList/Organisation/Organisation.module.css [hunk 2]
```diff
@@ -47,3 +47,9 @@
 .AddDayLabel {
   font-size: 16px;
 }
+
+@media (max-width:768px) {
+  .organization > div:nth-child(n + 2), .organization > div:first-child {
```
Comment (kurund) [inline src/containers/SettingList/Organisation/Organisation.module.css:52]: Let's discuss the approach. I feel it's better to have proper class selectors.

**Review by kurund — CHANGES_REQUESTED** [review]

**Comment by enigmaticmahesh** [comment]
@kurund I have made some changes as per the review. let me know for any changes if required

File: src/components/UI/Form/AutoComplete/AutoComplete.module.css [hunk 3]
```diff
@@ -47,3 +47,9 @@
   font-weight: 400;
   margin: 0 0 8px 16px;
 }
+
+@media (max-width:768px) {
```
Comment (kurund) [inline src/components/UI/Form/AutoComplete/AutoComplete.module.css:51]:
```suggestion
@media screen and (max-width: 768px) {
```

Can we please update media queries to include type in all places?

File: src/components/UI/Form/Calendar/Calendar.module.css [hunk 4]
```diff
@@ -7,3 +7,13 @@
 .CalendarInput .MuiOutlinedInput-adornedEnd {
   padding-right: 2px !important;
 }
+
+@media (max-width:768px) {
```
Comment (kurund) [inline src/components/UI/Form/Calendar/Calendar.module.css:11]: Do you have formatting on Save on? There should be space after `:`

File: src/containers/InteractiveMessage/InteractiveOptions/InteractiveOptions.module.css [hunk 5]
```diff
@@ -53,3 +53,12 @@
   composes: Translation from 'App.module.css';
   margin-bottom: 8px;
 }
+
+@media (max-width:768px) {
+  .RadioLabelWrapper:last-child {
+    margin-left: 0px;
+  }
+  .ForMobile {
```
Comment (kurund) [inline src/containers/InteractiveMessage/InteractiveOptions/InteractiveOptions.module.css:61]: It's good to have className reflecting where it's used etc. So in this case may be chang it to `RadioGroup`

File: src/containers/Trigger/Trigger.module.css [hunk 6]
```diff
@@ -22,3 +22,5 @@
   width: 29px;
   height: 29px;
 }
+
```
Comment (kurund) [inline src/containers/Trigger/Trigger.module.css:25]: Usually, VSCode automatically removes such extra spaces when you save the file if the format on saving is enabled.  Can you please check your editor settings?

**Review by kurund — CHANGES_REQUESTED** [review]

**Review by kurund — APPROVED** [review]
made changes as per feedback.

**Pull request merged**